Sources/SUIComponents/Atoms/Button+HMRC.swift

```diff
@@ -53,15 +53,20 @@ struct PrimaryButtonStyle: ButtonStyle {
 
 struct SecondaryButtonStyle: ButtonStyle {
     let padding: CGFloat
-    init(_ padding: CGFloat = .spacer16) {
+    let image: Image?
```

Comment:
As discussed - lets lose this until we make the [IconButtonView Component](https://jira.tools.tax.service.gov.uk/browse/HMA-4860)